includes/class-wc-google-gtag-js.php

```diff
@@ -296,37 +367,34 @@ public function product_detail( $product ) {
 	 * @param array $cart items/contents of the cart
 	 */
 	public function checkout_process( $cart ) {
-		$items = "[";
-
+		$items = array();
 		foreach ( $cart as $cart_item_key => $cart_item ) {
-			$product     = apply_filters( 'woocommerce_cart_item_product', $cart_item['data'], $cart_item, $cart_item_key );
-
-			$items .= "
-				{
-					'id': '" . self::get_product_identifier( $product ) . "',
-					'name': '" . esc_js( $product->get_title() ) . "',
-					'category': " . self::product_get_category_line( $product );
+			$product = apply_filters( 'woocommerce_cart_item_product', $cart_item['data'], $cart_item, $cart_item_key );
+
+			$item_data = array(
+				'id'       => self::get_product_identifier( $product ),
+				'name'     => $product->get_title(),
+				'category' => self::product_get_category_line( $product ),
+				'price'    => $product->get_price(),
+				'quantity' => $cart_item['quantity'],
+			);
 
-			$variant     = self::product_get_variant_line( $product );
+			$variant = self::product_get_variant_line( $product );
 			if ( '' !== $variant ) {
-				$items .= "
-					'variant': " . $variant;
+				$item_data['variant'] = $variant;
 			}
 
-			$items .= "
-					'price': '" . esc_js( $product->get_price() ) . "',
-					'quantity': '" . esc_js( $cart_item['quantity'] ) . "'
-				},";
+			$items[] += $item_data;
```

Comment:
If we are just appending to the array we don't need the `+` operator.

```suggestion
			$items[] = $item_data;
```